app/src/main/java/com/example/habitsmasher/ui/profile/ProfileFragment.java

```diff
@@ -49,34 +52,79 @@ public View onCreateView(@NonNull LayoutInflater inflater,
                              ViewGroup container, Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.fragment_profile, container, false);
         // create the sample user
-        User user = new User("1", "TestUser", "123@gmail.com", "123");
+        User user = new User("1", "TestUser", "123@gmail.com", "123", new ArrayList<String>(), new ArrayList<String>());
 
         SharedPreferences sharedPref = getContext().getSharedPreferences(USER_DATA_PREFERENCES_TAG, Context.MODE_PRIVATE);
         user.setUsername(sharedPref.getString(USERNAME_SHARED_PREF_TAG, "user"));
         String currentUserId = sharedPref.getString(USER_ID_SHARED_PREF_TAG, "id");
 
         // get the UI elements
         TextView usernameTextView = view.findViewById(R.id.username);
-        TextView numberOfFollowers = view.findViewById(R.id.number_followers);
-        TextView numberOfFollowing = view.findViewById(R.id.number_following);
+        Button numberOfFollowersButton = view.findViewById(R.id.number_followers);
```

Comment:
Changed the TextViews to Buttons, this can be undone and just have the text clickable. I think that this makes it very obvious that it is clickable however.

Reply:
I think I'd prefer if the buttons were white, with a slight outline instead of them being jarring blue on an all-white page. Just my personal preference.